### PR TITLE
Set LangVersion of all net6 csprojs to default, and bump versions of a few packages.

### PR DESCRIPTION
**Pull request by clin1234**
Perhaps we need a GH Action to automate updates of trivial packages...

### CODE REVIEW

**Comment by christophwille** [comment]
We very much intentionally do not have such an update task.

File: packages.props [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	-->
 	<PropertyGroup>
 		<SystemCollectionsImmutableVersion>6.0.0</SystemCollectionsImmutableVersion>
-		<SystemReflectionMetadataVersion>6.0.0</SystemReflectionMetadataVersion>
+		<SystemReflectionMetadataVersion>6.0.1</SystemReflectionMetadataVersion>
```
Comment (christophwille) [inline packages.props:9]: @siegfriedpammer I take we are intentionally not bumping this beyond 6.0.0 because ICS.D is using it (and thus Roslyn).

Comment (siegfriedpammer) [reply packages.props:9]: Yes, please revert this. Thanks!

Comment (christophwille) [reply packages.props]: With the exception of ICSharpCode.Decompiler.csproj - what is the point of the exercise anyways?

**Pull request closed without merging**